spec/stacktrace_spec.rb

```diff
@@ -25,6 +26,21 @@
     }
   end
 
+  it "includes bugsnag lines marked out of project" do
```

Comment:
I dont think this should be here. I think its a test of report and not stacktrace.